sqlglot/parser.py

```diff
@@ -1917,6 +1920,28 @@ def _parse_convert(self):
             to = None
         return self.expression(exp.Cast, this=this, to=to)
 
+    def _parse_substring(self):
+        args = [self._parse_term()]
+
+        # Postgres supports the form: substring(string [from int] [for int])
+        # https://www.postgresql.org/docs/9.1/functions-string.html @ Table 9-6
+
+        if self._match(TokenType.FROM):
+            args.append(self._parse_term())
+
+            if self._match(TokenType.FOR):
+                args.append(self._parse_term())
+        elif self._match(TokenType.COMMA):
+            if self._match(TokenType.R_PAREN):
```

Comment:
this is unnecessary let’s remove this check

Reply:
If removed, it recognizes `substring('bla',)` as valid sql. Is this ok?

Reply:
yes